frontend/assets/javascripts/src/modules/form/address/rules.js

```diff
@@ -22,22 +23,22 @@ define([
     var addRules = function (context) {
         var countrySelect = context.querySelector(COUNTRY_SELECTOR);
         var $countySelectParent = $(utilsHelper.getSpecifiedParent(context.querySelector(COUNTY_OR_STATE_SELECTOR), FORM_FIELD_CLASSNAME));
-        var $stateSelectParent = $(utilsHelper.getSpecifiedParent(context.querySelector(STATE_SELECTOR), FORM_FIELD_CLASSNAME));
-        var $provinceSelectParent = $(utilsHelper.getSpecifiedParent(context.querySelector(PROVINCE_SELECTOR), FORM_FIELD_CLASSNAME));
-
+        var $usStateSelectParent = $(utilsHelper.getSpecifiedParent(context.querySelector(US_STATE_SELECTOR), FORM_FIELD_CLASSNAME));
+        var $caProvinceSelectParent = $(utilsHelper.getSpecifiedParent(context.querySelector(CA_PROVINCE_SELECTOR), FORM_FIELD_CLASSNAME));
+        var $ausStateSelectParent = $(utilsHelper.getSpecifiedParent(context.querySelector(AUS_STATE_SELECTOR), FORM_FIELD_CLASSNAME));
         countrySelect.addEventListener('change', function (e) {
             var select = e && e.target;
-            implementRules(context, select, $countySelectParent, $stateSelectParent, $provinceSelectParent);
+            implementRules(context, select, $countySelectParent, $usStateSelectParent, $caProvinceSelectParent, $ausStateSelectParent);
         });
 
-        implementRules(context, countrySelect, $countySelectParent, $stateSelectParent, $provinceSelectParent);
+        implementRules(context, countrySelect, $countySelectParent, $usStateSelectParent, $caProvinceSelectParent, $ausStateSelectParent);
     };
 
-    var implementRules = function (context, select, $countySelectParent, $stateSelectParent, $provinceSelectParent) {
+    var implementRules = function (context, select, $countySelectParent, $stateSelectParent, $provinceSelectParent, $ausStateSelectParent) {
```

Comment:
Worth updating:

`$stateSelectParent` -> `$usStateSelectParent`
`$provinceSelectParent` -> `$caProvinceSelectParent`

for consistency with the changes above?